src/clj/puppetlabs/services/jruby/jruby_puppet_core.clj

```diff
@@ -192,140 +167,25 @@
       (update-in [:master-run-dir] #(or % default-master-run-dir))
       (update-in [:master-log-dir] #(or % default-master-log-dir))
       (update-in [:max-active-instances] #(or % (default-pool-size (ks/num-cpus))))
-      (update-in [:max-requests-per-instance] #(or % 0))
+      (update-in [:max-borrows-per-instance] #(or % 0))
```

Comment:
We want to keep this setting in puppet server's config as `max-requests-per-instance` (so that we don't have any breaking changes) - so we're at some point going to need to take it in as `max-requests-per-instance` but then associate the value to the config as `max-borrows-per-instance`.